src/presentational/TeacherForm.jsx

```diff
@@ -87,85 +93,82 @@ class TeacherForm extends Component {
           <p>Before you get started setting up your first classroom, please answer the following questions about how you plan to use WildCam Lab in your teaching.</p>
         </div>
         <form onSubmit={this.handleSubmit}>
+
           <div className="form-group">
-           <label>Where do you teach?</label>
-           <input className="form-control"
-            type="text"
-            name="country"
-            placeholder="Insert a country"
-            autofocus="true"
-            value={this.state.country}
-            required
-            onChange={this.handleChange}/>
+            <label>Where do you teach?</label>
```

Comment:
This label isn't associated with the dropdown. It might be better to make it part of the Dropdown component. (see above.)